Makefile

```diff
@@ -31,12 +33,13 @@ package_binary = $(COMMAND)$(BINARY_EXT_$(os))
 all: lint test package ## Run lint, test, build, and package
 
 PHONY: install-controller
```

Comment:
is "PHONY" a real label, as opposed to ".PHONY"? Or is this a typo in a previous merged commit by someone?

Reply:
Its in the prior code drop.  Will look into it.

Reply:
I changed it, for some reason it didn't seem to have any ill effects.  But to be consistent it should be .PHONY.